main.go

```diff
@@ -60,50 +61,52 @@ email, it contains potential sensitive information!
 `
 
 var modules = map[string]func(*collection.Collection){
+	"ansible":         ansible.Collect,
```

Comment:
Seems like all these fields were commited prematurely and are not related to redis?

Reply:
Just reordered to have it alphabetical.  
Not important enough to give it an own commit imo.